_posts/2018-06-29-upgrade-rails-from-5-0-to-5-1.markdown

```diff
@@ -116,6 +116,8 @@ already deprecated and covered in a [previous upgrade](https://fastruby.io/blog/
 been replaced by `#distinct`. Check out https://github.com/rails/rails/pull/20198
 for the discussion.
 
+- The ordering of association statements is now significant.  In particular, you can no longer declare a `has_many :through` association before declaring the association that implements the `through`.  You must declare the joined model first.
```

Comment:
@rdormer This makes sense. Could you add an example? Sometimes it is useful to show some code so people understand.